From ded2818b78150049e0cf5934eb4a752e1972e291 Mon Sep 17 00:00:00 2001
From: Sebastian Kayser <skayser@opencsw.org>
Date: Thu, 18 Aug 2011 00:02:54 +0200
Subject: [PATCH] Make stunnel.conf-sample.in honor --sysconfdir,
 --localstatedir

In case someone ./configured with a --sysconfdir or --localstatedir
rooted elsewhere than prefix, stunnel.conf-sample.in would have
disregarded it, because it only honors --prefix. This patch
introduces handling for the more specific --sysconfdir/--localstatedir.
---
 tools/stunnel.conf-sample.in |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/tools/stunnel.conf-sample.in b/tools/stunnel.conf-sample.in
index 37fea53..6a4ccc6 100644
--- a/tools/stunnel.conf-sample.in
+++ b/tools/stunnel.conf-sample.in
@@ -8,7 +8,7 @@
 
 ; A copy of some devices and system files is needed within the chroot jail
 ; Chroot conflicts with configuration file reload and many other features
-chroot = @prefix@/var/lib/stunnel/
+chroot = @localstatedir@/lib/stunnel/
 ; Chroot jail can be escaped if setuid option is not used
 setuid = nobody
 setgid = @DEFAULT_GROUP@
@@ -25,8 +25,8 @@ pid = /stunnel.pid
 ; *****************************************************************************
 
 ; Certificate/key is needed in server mode and optional in client mode
-cert = @prefix@/etc/stunnel/mail.pem
-;key = @prefix@/etc/stunnel/mail.pem
+cert = @sysconfdir@/stunnel/mail.pem
+;key = @sysconfdir@/stunnel/mail.pem
 
 ; Authentication stuff needs to be configured to prevent MITM attacks
 ; It is not enabled by default!
@@ -35,12 +35,12 @@ cert = @prefix@/etc/stunnel/mail.pem
 ; CApath is located inside chroot jail
 ;CApath = /certs
 ; It's often easier to use CAfile
-;CAfile = @prefix@/etc/stunnel/certs.pem
+;CAfile = @sysconfdir@/stunnel/certs.pem
 ; Don't forget to c_rehash CRLpath
 ; CRLpath is located inside chroot jail
 ;CRLpath = /crls
 ; Alternatively CRLfile can be used
-;CRLfile = @prefix@/etc/stunnel/crls.pem
+;CRLfile = @sysconfdir@/stunnel/crls.pem
 
 ; Disable support for insecure SSLv2 protocol
 options = NO_SSLv2
-- 
1.7.6

